README.md

```diff
@@ -1,4 +1,70 @@
-MolSim
+MolSim2024 - Group D
 ===
 
-The Molecular Dynamics teaching code.
+## Team members
+
+- Johannes Hupe
+- Julius Kramer
+- Tim Scholl
+
+
+## Prerequisites
+
+Developed and tested with these versions, other versions might work
+
+- clang 17.0.6
+- lldb 17.0.6
+- cmake 3.27.7
+- ninja 1.11.1
+
+These environment variables are assumed to be always set. They are used to select which environment should be build. 
+There purpose is simplifying the documentation only.
+
+```shell
+RELEASE_BUILD_DIR=build-release
+DEBUG_BUILD_DIR=build-debug
+
+BUILD_DIR=RELEASE_BUILD_DIR # or DEBUG_BUILD_DIR if debugging
+```
+
+Run this in the root folder of this repository
+
+```shell
+/usr/bin/cmake  -DCMAKE_BUILD_TYPE=Release \
+                -DCMAKE_MAKE_PROGRAM=/usr/bin/ninja \
+                -DCMAKE_CXX_COMPILER=/usr/bin/clang++ \
+                -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
+                -G Ninja \
+                -S $(pwd) \
+                -B $(pwd)/${BUILD_DIR}
+```
```

Comment:
I'm not so sure how I feel about the build instructions. They are not wrong, but very specific.
For example, these environment variables make everything sound more complicated than it is. I would just go with an example build folder called `build`. 
Also, for the  `cmake` command, I would just say `cmake` instead of specifying the full path, since this would not work if CMake is installed in a different place. Then `-S $(pwd)` is not really needed when you already say that this command should be executed in the root folder. And `-DCMAKE_EXPORT_COMPILE_COMMANDS=ON` is only needed for the `clang-tidy` analysis I believe.